client/src/cordova/android/OutlineAndroidLib/outline/src/main/java/org/outline/vpn/VpnTunnelService.java

```diff
@@ -228,7 +230,9 @@ private synchronized PlatformError startTunnel(
       this.tunFd = null;
     }
 
-    final NewClientResult clientResult = Outline.newClient(config.transportConfig);
+    final ClientConfig clientConfig = new ClientConfig();
+    clientConfig.setDataDir(this.getFilesDir().getAbsolutePath());
+    final NewClientResult clientResult = clientConfig.new_(config.id, config.transportConfig);
```

Comment:
*nit* `new_` looks ugly in Java, probably rename to `NewClient`? Or we can do the rename later.